content/git/concepts/clone/clone.md

```diff
@@ -14,22 +14,22 @@ CatalogContent:
 
 In Git, cloning means copying an existing remote repository to your current directory.
 
-Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on your computer. Here's a command that will help you interact with a remote branch &mdash; a branch on a server that can be accessed through the internet.
+Some of the basic Git commands have been performed only on a local branch, that is, a branch that exists on a local machine. Here's a command that will interact with a remote branch `&mdash`; a branch on a server that can be accessed through the internet.
```

Comment:
@KTom101 Is there a reason you removed the semicolon and added backticks around the em dash here? Is that a specific syntax for Codecademy's markdown renderer? Because as far as I know that won't be rendered correctly with standard markdown.